gadget/gadget.go

```diff
@@ -720,7 +721,7 @@ func validateRole(vs *VolumeStructure, vol *Volume) error {
 	}
 
 	switch vsRole {
-	case SystemData, SystemSeed:
+	case SystemData, SystemSeed, SystemSave:
```

Comment:
validateVolume and related helper also need consistency checks about SystemSave, for example it should exist only if data and seed exist,  it should not a label but only the implicit one and maybe other things ... ?